lib/afdc/mailchimp.rb

```diff
@@ -0,0 +1,11 @@
+module Mailchimp
+  def subscribe_to_mailchimp(list_id, user)
+    gibbon = Gibbon::Request.new(api_key: ENV['MAILCHIMP_ID'])
```

Comment:
Does it make sense to change the name of this constant to `MAILCHIMP_API_KEY`?

Reply:
Yeah, that would be good, changing it now.